operators/dell-csi-operator-certified/1.7.0/manifests/dell-csi-operator-certified.clusterserviceversion.yaml

```diff
@@ -0,0 +1,2040 @@
+                  {
+                    "name": "X_CSI_POWERMAX_PORTGROUPS",
+                    "value": ""
+                  },
+                  {
+                    "name": "X_CSI_TRANSPORT_PROTOCOL",
+                    "value": ""
+                  },
+                  {
+                    "name": "X_CSI_POWERMAX_PROXY_SERVICE_NAME",
+                    "value": ""
+                  },
+                  {
+                    "name": "X_CSI_GRPC_MAX_THREADS",
+                    "value": "4"
+                  }
+                ],
+                "image": "docker.io/dellemc/csi-powermax@sha256:2fc83456ec2fb2c0336725e039a3db699a0202468cd8e3fd12d7ef2423068fc8",
+                "imagePullPolicy": "IfNotPresent"
+              },
+              "configVersion": "v2.2.0",
+              "dnsPolicy": "ClusterFirstWithHostNet",
+              "forceUpdate": false,
+              "node": {
+                "envs": [
+                  {
+                    "name": "X_CSI_POWERMAX_ISCSI_ENABLE_CHAP",
+                    "value": "false"
+                  }
+                ]
+              },
+              "replicas": 2
+            }
+          }
+        },
+        {
+          "apiVersion": "storage.dell.com/v1",
+          "kind": "CSIPowerMaxRevProxy",
+          "metadata": {
+            "name": "powermax-reverseproxy",
+            "namespace": "test-powermax"
+          },
+          "spec": {
+            "config": {
+              "linkConfig": {
+                "backup": {
+                  "skipCertificateValidation": true,
+                  "url": "https://0.0.0.0:8443"
+                },
+                "primary": {
+                  "certSecret": "",
+                  "skipCertificateValidation": true,
+                  "url": "https://0.0.0.0:8443"
+                }
+              },
+              "mode": "Linked"
+            },
+            "image": "docker.io/dellemc/csipowermax-reverseproxy@sha256:c236cb87d5799ef239b3e5df956761e1c042f3d5686c8e5728c06edfef3095b7",
+            "imagePullPolicy": "IfNotPresent",
+            "tlsSecret": "csirevproxy-tls-secret"
+          }
+        },
+        {
+          "apiVersion": "storage.dell.com/v1",
+          "kind": "CSIPowerStore",
+          "metadata": {
+            "name": "test-powerstore",
+            "namespace": "test-powerstore"
+          },
+          "spec": {
+            "driver": {
+              "common": {
+                "envs": [
+                  {
+                    "name": "X_CSI_POWERSTORE_NODE_NAME_PREFIX",
+                    "value": "csi"
+                  },
+                  {
+                    "name": "X_CSI_FC_PORTS_FILTER_FILE_PATH",
+                    "value": "/etc/fc-ports-filter"
+                  }
+                ],
+                "image": "docker.io/dellemc/csi-powerstore@sha256:08c5effefadf2e55103eb49ef2a5a325d9904e7391b6eeb72701dae6e682ad9e",
+                "imagePullPolicy": "IfNotPresent
```

Comment:
@rensyct There is a missing `"` here.

Reply:
Thank you @acornett21 for pointing out the error in CSV